content/docs/studio/create-view.md

```diff
@@ -0,0 +1,54 @@
+# Create a view
+
+To create a new view for your repository, follow these steps.
+
+1. Click on add a View. All the organizations that you have access to will be
+   listed. ![](/img/studio/create_view.png)
+
+2. Open the organization whose repository you want to connect to. You can also
+   use the search bar to search directly by the name of the repo to which you
+   want to connect. ![](/img/studio/select_repo.png)
+
+3. Select the Git repository for which you want to create a view. For this,
+   hover over the required repository and click on Connect.
+
+4. Once you click on connect, you will be able to specify advanced settings for
+   the connection. ![](/img/studio/view_settings.png)
+
+In most scenarios, you can skip the advanced settings. Refer to the 'Advanced
+Settings' section to understand when you need to specify these settings.
```

Comment:
@jorgeorpinel is this the correct format for asides/notes? Not sure about emoji conventions :)

```suggestion
> :point_right: In most scenarios, you can skip the advanced settings. Refer to
> the [Advanced Settings](#advanced-settings) section below for more details.
```